### ISSUE TITLE
Mensa opening hours are wrong/outdated

### ISSUE THREAD

**Issue by ManuelHu** [TUM-Dev/Campus-Android#1440]
## Expected Result

The opening hours of the mensa matches the ones specified by the Studentenwerk. So for Garching, the opening hours [shoukd be](https://www.studentenwerk-muenchen.de/mensa/standorte-und-oeffnungszeiten/garching/) 11:00-14:00 on Monday till Friday.

## Actual Result

Opening hours 
```
Mo-Do 11-16, Fr 11-13:45\nvorl.freie Zeit
Mo-Do 11-14\nFr11-13:45
```
Also in the menu view, the wrong end time (16:00) is used for calculations.

## Problem

The opening hours are outdated (at least since some semesters, I suppose...) and do not match the real ones...

### Environment
* OS version: Android 12
* TUM Campus App version: most recent from google play (3.13)
* Language: German

**Comment by kmodexc** [comment]
Seems to be a backend issue:
```
{
    "id": "66",
    "category": "cafeteria_gar",
    "name": "Mensa Garching",
    "address": "Lichtenbergstr. 2, 85748 Garching",
    "room": "",
    "transport_station": "",
    "opening_hours": "Mon-Thu 11-16, Fri 11-13:45\nLecture free period:\nMon-Thu 11-14 Fri 11-13:45",
    "infos": "Payment: Legic-Card",
    "url": "https://www.studentenwerk-muenchen.de/mensa/standorte-und-oeffnungszeiten/garching/#c1430",
    "language": "en",
    "reference_id": "422"
  }
```

**Comment by kmodexc** [comment]
I will create an eat-api client.

**Comment by CommanderStorm** [comment]
> I will create an eat-api client.

you should have a look at https://github.com/TUM-Dev/Campus-Android/pull/1422 and what the status of this PR is.
There is already a lot of work, that has been done by @Liqs-v2 

Woulld be a shame to do the same work twice..

**Comment by Liqs-v2** [comment]
@kmodexc I would wait for work on the cafeteria part until the eat-api PR is merged. It will still not be fully done yet, as that is a large section of the app, but that issue might just fix itsself then.

**Comment by joschahenningsen** [comment]
I changed the opening hours on the backend, thanks reporting, feel free to ping me if any other times are off :)

**Comment by ManuelHu** [comment]
Thanks for updating the opening hours, @joschahenningsen . Also the Espressobar and all other StuCafes in Garching are not correct.

Also the addresses are outdated, the C2 Campus-Cneipe, Mensa Garching, Mensa Garching Infopoint & StuCafé Mensa Garching are now at Boltzmannstraße 19 instead of Lichtenbergstraße 2, which is the old mensa building...

I did not check the other campuses, but I suspect the opening ours also to be outdated.

**Comment by Bentipa** [comment]
Is this going to be fixed through the eat-api? @joschahenningsen

**Comment by kordianbruck** [comment]
Yes, most likely. Eat-api has newer information and we can throw away the old obsolete stuff then
